Frameworks/Foundation/NSSet.mm

```diff
@@ -526,6 +526,9 @@ - (NSArray*)sortedArrayUsingDescriptors:(NSArray*)descriptors {
  @Status Interoperable
 */
 - (id)member:(id)obj {
+    if (obj == nil) {
+        return nil;
+    }
```

Comment:
Insufficient. You should make sure that nil can't be added to the set. That more closely matches the reference platform and then this check would not be needed.